_includes/footer.html

```diff
@@ -127,12 +127,7 @@
   l.href = '/css/main.css';
```

Comment:
This should be moved to `head`.

Reply:
In the head section, we only have critical CSS. The rest is in the footer to optimize the page load time.